Source/Core/VideoCommon/AsyncShaderCompiler.cpp

```diff
@@ -68,6 +68,19 @@ bool AsyncShaderCompiler::HasCompletedWork()
   return !m_completed_work.empty();
 }
 
+void AsyncShaderCompiler::ClearAllWork()
+{
+  {
+    std::lock_guard<std::mutex> guard(m_pending_work_lock);
+    m_pending_work.clear();
+  }
+
+  {
+    std::lock_guard<std::mutex> guard(m_completed_work_lock);
```

Comment:
```suggestion
    std::lock_guard guard(m_pending_work_lock);
    m_pending_work.clear();
  }

  {
    std::lock_guard guard(m_completed_work_lock);
```
I suppose you're copying the adjacent code, but I'd probably just rely on "CTAD". We do this for most of our other locks. It's less noisy to read and creates less future diff noise if the mutex type needs to change to recursive_mutex or whatever.